flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -286,7 +287,7 @@ private void commitPendingResult(
       }
       continuousEmptyCheckpoints = 0;
     } else {
-      LOG.info("Skipping committing empty checkpoint {}", checkpointId);
```

Comment:
it seems that this log line was added in 1.15 only and wasn't ported to 1.14 or 1.16